sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -23,6 +23,14 @@ export interface AdministratorContact {
     phone?: string;
 }
 
+// @public
+export interface AdministratorDetails {
+    email?: string;
```

Comment:
I think I saw emailAddress and phoneNumber elsewhere. Just double check that these names are aligned :)

Reply:
This property is `email` in the other language implementations of the keyvault-certificates package.

Reply:
I'm going to merge this PR. Please let me know if I should follow up with anything else. I appreciate the feedback!